### ISSUE TITLE
关于外生变量嵌入问题

### ISSUE THREAD

**Issue by s1h3ibai** [thuml/TimeXer#13]
def forward(self, x, x_mark):
        x = x.permute(0, 2, 1)
        # x: [Batch Variate Time]
        if x_mark is None:
            x = self.value_embedding(x)
        else:
            x = self.value_embedding(torch.cat([x, x_mark.permute(0, 2, 1)], 1))
        # x: [Batch Variate d_model]
        return self.dropout(x)
按照论文中的说法，有C个外生变量就嵌入C个token，但是代码中通过cat将x和 x_mark进行拼接导致形成了2C个Token, 请问为什么要这样操作呢？  我将self.value_embedding(torch.cat([x, x_mark.permute(0, 2, 1)], 1))修改成self.value_embedding(x)，性能还得到了一点提升

**Comment by wang-lucy** [comment]
您好，x_mark是4个时间戳外生变量，生成方式请参考https://github.com/thuml/TimeXer/blob/main/data_provider/data_loader.py#L72。将时间信息作为变量的处理方式是follow iTransformer的。
